requirements/base.txt

```diff
@@ -1,13 +1,13 @@
 boto==2.42.0                        # MIT
-Django==1.10.7 						# BSD License
-django-countries==4.0               # MIT
-djangorestframework==3.5.3			# BSD
-django-rest-swagger==0.3.8  		# BSD
-djangorestframework-csv==1.4.1		# BSD
-django-storages==1.5.1              # BSD
+Django==1.11 						# BSD License
+django-countries==4.5               # MIT
+djangorestframework==3.6.2			# BSD
+django-rest-swagger==2.1.2  		# BSD
+djangorestframework-csv==2.0.0		# BSD
+django-storages==1.5.2              # BSD
 elasticsearch-dsl==0.0.11           # Apache 2.0
-ordered-set==2.0.1                  # MIT
-tqdm==4.10.0                        # MIT
+ordered-set==2.0.2                  # MIT
+tqdm==4.11.2                        # MIT
```

Comment:
Thanks for updating all these :+1:

I think it might also be a good idea to take this opportunity to update the reqs in `requirements/text.txt` too. At the very least the Django-related ones in case they've been updated for 1.11.